src/main/java/io/blockfrost/sdk/api/model/Script.java

```diff
@@ -0,0 +1,29 @@
+package io.blockfrost.sdk.api.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+/**
+ * Script
+ */
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
```

Comment:
@bentaye As you are using @Data annotation here, I think you don't need @EqualsAndHashCode and @ToString . These two are already included in @Data

You may want to remove these additional annotations or use @Getter and keep these two annotations. Btw, Getter should be enough as these are immutable objects. But I understand we are using Data in other model class files and you may want to keep it for consistency.

Reply:
Agreed, replaced `@Data` with `@Getter` in all classes I added. 
And I think it should be the same in pretty much all the model classes.

Reply:
Thanks. Yes, we should plan to change other model class later.